index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="./style.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Lato:wght@300;400;700;900&family=Montserrat:wght@400;700&family=Playfair+Display:wght@400;500&family=Poppins:wght@100;300;400&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <ul>
+        <li><img src="icons/toolbar/facebook.png" alt="facebook-ribbon-icon"></li>
+        <li><img src="icons/toolbar/twitter-sign.png" alt="twitter-ribbon-icon"></li>
+        <li><select name="language" id="language">
+          <option value="english">English</option>
+          <option value="swahili">Kiswahili</option>
+        </select></li>
+        <li>My Page</li>
+        <li>Login</li>
+      </ul>
+    </div>
+
+    <nav class="navbar">
+      <a href="index.html"><img class="logo" src="icons/MotoImg/Moto_Gp_logo.svg.png" alt="MotoGP"></a>
+      <ul class="nav-menu">
+          <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+          <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+          <li class="nav-item"><a href="#drivers" class="nav-link">Riders and Teams</a></li>
+          <li class="nav-item"><a href="#races.html" class="nav-link">Races</a></li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+
+      <ul class="desktop-nav">
+          <li><a href="about.html" class="page-links-1">About</a></li>
+          <li><a href="#drivers" class="page-links">Our Riders</a></li>
+          <li><a href="#race-days" class="page-links">Races</a></li>
+          <li><a href="#https://www.formula1.com/en/latest.html" class="page-links">News</a></li>
+          <li><a href="https://www.formula1.com/en/latest/tags.race-calendar.2Sa4DEdBSUMcOqa8aYceEe.html" class="page-links campaign" id="campaign">F1 Campaign</a></li>
+      </ul>
+    </nav>
```

Comment:
[Optional] It'd be nice to adjust the spacing between the nav items as your design is quite different from the design given in that aspect. The spacing between your nav items is alot and not so nice aesthetically 😄 . Also your nav color for some of the items is blue,which is different from the color in the design

Your design :

![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/20537712/fd34eb99-c0f5-4dca-a273-0c658e9be907)

Figma design :
![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/20537712/5598a9e6-81fb-4a1d-a28b-de152fe3d610)